xstate-chatbot/nodejs/src/machine/hospital-details.js

```diff
@@ -0,0 +1,1068 @@
+const { assign, actions } = require('xstate');
+const moment = require('moment');
+const dialog = require('./util/dialog');
+const mediaUtil = require('./util/media');
+const config = require('../env-variables');
+const { messages, grammers } = require('./messages/hospital-details');
+const { personService, bedsService } = require('./service/service-loader');
+
+const hospitalFlow = {
+  id: 'hospitalFlow',
+  initial: 'nodalOfficer',
+  onEntry: assign((context, event) => {
+    context.slots.hospital = {};
+    context.slots.previoushospitaldata = {};
+  }),
+  states: {
+    nodalOfficer: {
+      id: 'nodalOfficer',
+      initial: 'process',
+      states: {
+        prompt: {
+          onEntry: assign((context, event) => {
+          }),
```

Comment:
send message to user

Reply:
After choosing 5th option, hospital type is fetched by using user mobile number. So No message is required here to send.So removed prompt

Reply:
If it is a one-way message to the user, like in the case of informational messages in covid-info.js, then there is no need to make substates prompt, process, error. Refer to covid-info.js

Reply:
in covid-info.js, we have to select any one input then we can proceed but no such question is asked in hospital-details . It is in continue process

Reply:
I am telling you to see [this](https://github.com/egovernments/core-services/blob/869fbe1ff6f13796bafa37b765e8a55228d097d6/xstate-chatbot/nodejs/src/machine/covid-info.js#L78) state. It hasn’t been divided into 3 sub states.

Reply:
In nodalofficer state ,one process is there where response is fetched by using mobile number of nodal officer(these are registered nodal officer) and if response is '1' even then process continue otherwise an error message (No Nodal Officer Associated with this number) has been display.And i removed onentry of nodal officer state (code committed)